### PR TITLE
fix: replace containers/image with google/go-containerregistry

### PR DESCRIPTION
**Pull request by knqyf263**
Ref. https://github.com/aquasecurity/fanal/pull/96

### CODE REVIEW

File: rpc/common/service.pb.go [hunk 1]
```diff
@@ -366,7 +366,7 @@ type Vulnerability struct {
 	Description          string   `protobuf:"bytes,6,opt,name=description,proto3" json:"description,omitempty"`
 	Severity             Severity `protobuf:"varint,7,opt,name=severity,proto3,enum=trivy.common.Severity" json:"severity,omitempty"`
 	References           []string `protobuf:"bytes,8,rep,name=references,proto3" json:"references,omitempty"`
-	LayerId              string   `protobuf:"bytes,9,opt,name=layer_id,json=layerId,proto3" json:"layer_id,omitempty"`
+	Layer                *Layer   `protobuf:"bytes,10,opt,name=layer,proto3" json:"layer,omitempty"`
```
Comment (simar7) [inline rpc/common/service.pb.go:369]: I'm just curious: Why is this by reference? Are custom types always passed by reference?

File: pkg/detector/ospkg/debian/debian.go [hunk 2]
```diff
@@ -102,8 +102,7 @@ func (s *Scanner) Detect(osVer string, pkgs []ftypes.Package) ([]types.DetectedV
 				VulnerabilityID:  adv.VulnerabilityID,
 				PkgName:          pkg.Name,
 				InstalledVersion: installed,
-				//FixedVersion: adv.FixedVersion, // TODO: Why is this missing?
```
Comment (simar7) [inline pkg/detector/ospkg/debian/debian.go:105]: Would you know why is this missing?

File: rpc/common/service.pb.go (hunk 1 above)
Comment (knqyf263) [reply rpc/common/service.pb.go:369]: I'm not sure, but other custom types are also references.
https://github.com/aquasecurity/trivy/blob/1b112c30ca48cb2fbf5750f428665eea624d4404/rpc/cache/service.pb.go#L32

File: pkg/detector/ospkg/debian/debian.go (hunk 2 above)
Comment (knqyf263) [reply pkg/detector/ospkg/debian/debian.go:105]: This is because this bucket stores only unfixed vulnerabilities. I'll add a comment.

**Pull request merged**